recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,166 @@
+  -DBUILD_SHARED_LIBS=ON                       \
+  -DEIGEN_INCLUDE_DIR=${PREFIX}/include/eigen3 \
+  ..
+make -j${CPU_COUNT} install
+
+# fgr
+cd $SRC_DIR/FastGlobalRegistration
+FGR_SOURCE_DIR=$(pwd)/source
+mkdir -p build && cd build
+INC_FLAGS="-I${PREFIX}/include/eigen3 -I${PREFIX}/include -O3 -L${PREFIX}/lib -lflann_cpp -llz4 -O3 -std=c++11"
+cmake                                        \
+  ${CMAKE_ARGS}                              \
+  -DCMAKE_BUILD_TYPE=Release                 \
+  -DCMAKE_CXX_FLAGS="${INC_FLAGS}"           \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX}      \
+  -DCMAKE_VERBOSE_MAKEFILE=ON                \
+  -DFastGlobalRegistration_LINK_MODE=SHARED  \
+  ${FGR_SOURCE_DIR}
+make -j${CPU_COUNT}
+# Install
+FGR_INC_DIR=${PREFIX}/include/FastGlobalRegistration
+mkdir -p ${FGR_INC_DIR}
+cp ${FGR_SOURCE_DIR}/FastGlobalRegistration/app.h ${FGR_INC_DIR}
+FGR_LIB_DIR=${PREFIX}/lib
+mkdir -p ${FGR_LIB_DIR}
+cp FastGlobalRegistration/libFastGlobalRegistrationLib* ${FGR_LIB_DIR}
+
+# s2p
+cd $SRC_DIR/s2p
+baseDir=$(pwd)
+export CFLAGS="-I$PREFIX/include -O3 -DNDEBUG -march=native"
+export LDFLAGS="-L$PREFIX/lib"
+# Build the desired programs
+cd 3rdparty/mgm
+perl -pi -e "s#CFLAGS=#CFLAGS=$CFLAGS #g" Makefile
+perl -pi -e "s#LDFLAGS=#LDFLAGS=$LDFLAGS #g" Makefile 
+make -j${CPU_COUNT}
+cd $baseDir
+# msmw
+cd 3rdparty/msmw
+mkdir -p build
+cd build
+cmake ..                                                       \
+    ${CMAKE_ARGS}                                              \
+    -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_CXX_FLAGS="$CFLAGS"      \
+    -DPNG_LIBRARY_RELEASE="${PREFIX}/lib/libpng${SHLIB_EXT}"   \
+    -DTIFF_LIBRARY_RELEASE="${PREFIX}/lib/libtiff${SHLIB_EXT}" \
+    -DZLIB_LIBRARY_RELEASE="${PREFIX}/lib/libz${SHLIB_EXT}"    \
+    -DJPEG_LIBRARY="${PREFIX}/lib/libjpeg${SHLIB_EXT}"
+make -j${CPU_COUNT}
+cd $baseDir
+# msmw2
+cd 3rdparty/msmw2
+mkdir -p build
+cd build
+cmake ..                                                       \
+    ${CMAKE_ARGS}                                              \
+    -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_CXX_FLAGS="$CFLAGS"      \
+    -DPNG_LIBRARY_RELEASE="${PREFIX}/lib/libpng${SHLIB_EXT}"   \
+    -DTIFF_LIBRARY_RELEASE="${PREFIX}/lib/libtiff${SHLIB_EXT}" \
+    -DZLIB_LIBRARY_RELEASE="${PREFIX}/lib/libz${SHLIB_EXT}"    \
+    -DJPEG_LIBRARY="${PREFIX}/lib/libjpeg${SHLIB_EXT}"
+make -j${CPU_COUNT}
+cd $baseDir
+# Install the desired programs
+BIN_DIR=${PREFIX}/plugins/stereo/mgm/bin
+mkdir -p ${BIN_DIR}
+cp 3rdparty/mgm/mgm ${BIN_DIR}
+BIN_DIR=${PREFIX}/plugins/stereo/msmw/bin
+mkdir -p ${BIN_DIR}
+cp 3rdparty/msmw/build/libstereo/iip_stereo_correlation_multi_win2 \
+   ${BIN_DIR}/msmw
+BIN_DIR=${PREFIX}/plugins/stereo/msmw2/bin
+mkdir -p ${BIN_DIR}
+cp 3rdparty/msmw2/build/libstereo_newversion/iip_stereo_correlation_multi_win2_newversion \
+   ${BIN_DIR}/msmw2
+
+# libelas
+if [[ "$target_platform" == "aarch64" || "$target_platform" == "arm64" ]]; then
```

Comment:
I don't think these will ever be true, since `$target_platform` takes the form `{OS}-{ARCH}`. You might need a glob or something like that.